CHANGELOG.md

```diff
@@ -1,20 +1,24 @@
 # Changelog
 
-## Unreleased
+## 1.15.0
 
 ### Added
-- Enforcing a redirect to setup of otp device when none available for user [#550](https://github.com/jazzband/django-two-factor-auth/pull/500)
+- Enforcing a redirect to setup of otp device when none available for user (#499)
```

Comment:
Should this refer to PR #500? Also, isn't this a breaking change? It seems that with this addition it's not longer possible for optional 2FA (i.e. all users are now required to setup a 2FA device).

Reply:
I think I misunderstood the purpose of that PR, seems everything is working as before